actors/builtin/miner/miner_state.go

```diff
@@ -135,6 +135,20 @@ type SectorOnChainInfo struct {
 	VerifiedDealWeight abi.DealWeight // Integral of active verified deals over sector lifetime
 }
 
+// SealVerifyParams is the structure of information that must be sent with a
+// message to commit a sector. Most of this information is not needed in the
+// state tree but will be verified in sm.CommitSector. See SealCommitment for
+// data stored on the state tree for each sector.
+type SealVerifyParams struct {
```

Comment:
Since this ends up not being the params to an exported actor method or stored in state, but just an internal blob of data, please:
- rename to SealVerifyStuff
- move to miner_actor.go (perhaps just above the `verifySeal` helper that uses it)
- remove cbor-gen serialization

Maybe we can refactor to make the naming less stuffy later.

Reply:
> Maybe we can refactor to make the naming less stuffy later.

Sounds good.